README.rst

```diff
@@ -481,6 +481,20 @@ Removes module-level ``default_app_config`` assignments from ``__init__.py`` fil
 
     -default_app_config = 'my_app.apps.AppConfig'
 
+``BaseCommand.requires_system_checks``
```

Comment:
I think I tried to keep the fixer docs in the same order as the deprecation docs, so this would be before `EmailValidator`